ARCs/arc-0004.md

```diff
@@ -218,18 +239,19 @@ For example:
 
 ### Contracts
 
-A Contract is the complete set of methods that an app implements. It
+A contract is the complete set of methods that an app implements. It
```

Comment:
Why lowercase? Capitalized elsewhere and I like it as a proper noun

Reply:
Yes, there is a bit of inconsistency, because "interface" was always lower case. So not sure what convention to adopt.

Reply:
I'd support capitalizing Interface and Contract everywhere for consistency

Reply:
I've modified this. Doing so, I've realized there may be some slightly confusing naming.
Let me know what you think. I see two types of objects:
* a "smart contract" or "application" or "app" or "stateful application" or "contract" is an actual application on Algorand
* a "Contract" is "the complete set of methods that an app implements"

Maybe to clarify should we restrict the vocabulary to:
* "application" for the one above
* "Contract" for the one below

so no "contract call transaction" but "application call transaction".

What do you think?